html-syntax/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+  <html lang="en">
+
+    <head>
+      <title> HTML SYNTAX </title>
+    </head>
+
+    <body>
+      <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png" >
+      <h3> Pikachu </h3>
+      &copy; Pokemon. &reg; Nintendo.
```

Comment:
These should be wrapped in some sort of element, not just out in the open. Put them inside a `<p>` or something